modules/appeals_api/config/schemas/v2/200996.json

```diff
@@ -0,0 +1,214 @@
+    },
+    "nonBlankStringMaxLength140": {
+      "allOf": [ { "$ref": "#/definitions/nonBlankString" }, { "maxLength": 140 } ]
+    },
+    "date": {
+      "type": "string",
+      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+    },
+
+
+    "hlrCreate": {
+      "type": "object",
+      "properties": {
+        "data":     { "$ref": "#/definitions/hlrCreateData" },
+        "included": { "$ref": "#/definitions/hlrCreateIncluded" }
+      },
+      "additionalProperties": false,
+      "required": ["data", "included"]
+    },
+
+
+    "hlrCreateData": {
+      "type": "object",
+      "properties": {
+        "type":       { "type": "string", "enum": ["higherLevelReview"] },
+        "attributes": { "$ref": "#/definitions/hlrCreateDataAttributes" }
+      },
+      "additionalProperties": false,
+      "required": [ "type", "attributes" ]
+    },
+
+
+    "hlrCreateDataAttributes": {
+      "description": "If informal conference requested (`informalConference: true`), contact and time must be specified (`informalConferenceTime`). Explanation of abbreviated benefit types: `pension`: Pension / Survivor's Benefits, `voc_rehab`: Vocational Rehabilitation and Employment, `vha`: Veterans Health Administration, `nca`: National Cemetery Administration",
+      "oneOf": [
+        {
+          "type": "object",
+          "properties": {
+            "informalConference": { "type": "boolean", "enum": [false] },
+            "benefitType":        { "$ref": "#/definitions/hlrCreateBenefitType" },
+            "veteran":            { "$ref": "#/definitions/hlrCreateVeteran" },
+            "ssoSsocOptIn":       { "type": "boolean" }
+          },
+          "additionalProperties": false,
+          "required": [
+            "informalConference",
+            "sameOffice",
+            "benefitType",
+            "veteran"
+          ]
+        },
+        {
+          "$comment": "if informalConference requested, desired contact and time must be included",
+          "type": "object",
+          "properties": {
+            "informalConference":                 { "type": "boolean", "enum": [true] },
+            "sameOffice":                         { "type": "boolean" },
+            "benefitType":                        { "$ref": "#/definitions/hlrCreateBenefitType" },
+            "veteran":                            { "$ref": "#/definitions/hlrCreateVeteran" },
+            "hltCreateInformalConferenceContact": { "$ref": "#/definitions/hlrCreateInformalConferenceContact" },
+            "informalConferenceTime":             { "$ref": "#/definitions/hlrCreateInformalConferenceTime" },
+            "informalConferenceRep":              { "$ref": "#/definitions/hlrCreateInformalConferenceRep" }
+          },
+          "additionalProperties": false,
+          "required": [
+            "informalConference",
+            "sameOffice",
+            "benefitType",
+            "veteran",
+            "hltCreateInformalConferenceContact",
```

Comment:
Typo here and on line 71?

Reply:
Yes! I thought I fixed that. Thanks for the catch!